lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -140,9 +155,32 @@ assembleApkRelease:
   artifacts:
     paths:
       - $DEPLOY_RELEASE_APK_PATH
+      - $DEPLOY_RELEASE_AAR_PATH
+
+deployArtifactorySnapshot:
+  stage: deploy
+  rules: *snapshot_rules
+  script:
+    - ./gradlew tasks | grep publish -q
+    - if [ $? -eq 0 ]; then
+        ./gradlew publish -PartifactoryUsername=$CTI_USERNAME -PartifactoryPassword=$CTI_PASSWORD -PartifactoryURL="${ARTIFACTORY_SNAPSHOT_URL}"
```

Comment:
Nit but we want these templates to be used on other customer's Gitlabs, not just CTI Gitlabs. Within CTI's Gitlab I just added variables off `ARTIFACT_REPO_USERNAME` and `ARTIFACT_REPO_PASSWORD` which map to the previous `CTI_USERNAME` and `CTI_PASSWORD`. Can you please move this code to `ARTIFACT_REPO_USERNAME`, `ARTIFACT_REPO_PASSWORD`, and `ARTIFACT_SNAPSHOT_URL`?